src/components/Home/Hero/index.tsx

```diff
@@ -1,14 +1,17 @@
 import React from "react";
+import MaxWithBgColorContainer from "../../Layout/Layout/MaxWithBgColorContainer";
 
 import { HomeHeroHeading } from "./heroHeading";
 import { HomeHeroImage } from "./heroImage";
 
 const HomeHeroComponent = () => {
   return (
-    <main className="flex flex-col lg:flex-row flex-grow px-8 py-8 lg:items-center min-h-full">
-      <HomeHeroHeading />
-      <HomeHeroImage />
-    </main>
+    <MaxWithBgColorContainer>
+      <main className="flex flex-col lg:flex-row flex-grow px-8 py-8 lg:items-center min-h-full">
```

Comment:
How about this container under the main? It would be semantically more valid

Could it work well?

Reply:
![image](https://github.com/mrruby/get-bold/assets/103185922/e17857ab-b095-40e4-bb62-1972f23add17)

Reply:
ok, let's leave it as it is